UiBasic/ComponentsKT/ComponentsKT/ViewControllers/FormViewController.swift

```diff
@@ -39,105 +38,105 @@ class FormViewController: UIViewController {
     
-    // MARK: CUSTOMIZATION
+    deinit {
+        // Remove keyboard notification observers
+        NotificationCenter.default.removeObserver(self)
+    }
+    
+}
+
+// MARK: CUSTOMIZATION METHODS
+extension FormViewController {
     private func configureUI() {
-        // txtFirstName.translatesAutoresizingMaskIntoConstraints = false
         navigationView.layer.cornerRadius = 8
         mainView.layer.cornerRadius = 35
         mainView.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
         btnSaveInfo.isHidden = true
-        //btnSaveInfo.layer.cornerRadius = btnSaveInfo.bounds.height/2
     }
     
     private func configureTextFields() {
-        txtFirstName.setSemiBoldBlackPlaceholderText("First Name")
-        txtLastName.setSemiBoldBlackPlaceholderText("Last Name")
-        txtEmail.setSemiBoldBlackPlaceholderText("Email")
-        txtJobTitle.setSemiBoldBlackPlaceholderText("Job Title")
-        txtCompanyName.setSemiBoldBlackPlaceholderText("Company Name")
-
+        txtFirstName.setSemiBoldBlackPlaceholderText(Constants.Strings.txtFirstNameStr)
+        txtLastName.setSemiBoldBlackPlaceholderText(Constants.Strings.txtLastNameStr)
+        txtEmail.setSemiBoldBlackPlaceholderText(Constants.Strings.txtEmailStr)
+        txtJobTitle.setSemiBoldBlackPlaceholderText(Constants.Strings.txtJobTitleStr)
+        txtCompanyName.setSemiBoldBlackPlaceholderText(Constants.Strings.txtCompanyNameStr)
+        
         // Set delegate for each text field
         txtFirstName.delegate = self
         txtLastName.delegate = self
         txtEmail.delegate = self
         txtJobTitle.delegate = self
         txtCompanyName.delegate = self
-
+        
     }
     
     private func placeImageOnProfile() {
         let childView = UIView()
-        childView.frame = CGRect(x: (imgProfileView.bounds.width) - 22, y: imgProfileView.bounds.height - 25, width: 28, height: 28)
-        childView.backgroundColor = UIColor(named: "emailColor")
-        childView.layer.cornerRadius = childView.bounds.width/2
-        let cameraImg = UIImageView()
+        childView.frame = CGRect(x: (imgProfileView.bounds.width) - 20, y: imgProfileView.bounds.height - 20, width: 28, height: 28)
+        childView.backgroundColor = UIColor(named: Constants.Colors.headerThemeColor)
+        childView.layer.cornerRadius = childView.bounds.width / 2
         
-        cameraImg.image = UIImage(named: "camera")
+        let cameraImg = UIImageView()
+        cameraImg.image = UIImage(named: Constants.Strings.imgCameraStr)
         cameraImg.frame = CGRect(x: childView.bounds.midX - 6.5, y: childView.bounds.midY - 6.5, width: 13, height: 13)
         childView.layer.borderWidth = 2
         childView.layer.borderColor = UIColor.white.cgColor
         cameraImg.layer.cornerRadius = cameraImg.bounds.width / 2
-
         childView.clipsToBounds = true
         cameraImg.isUserInteractionEnabled = true
         childView.isUserInteractionEnabled = true
```

Comment:
Why isUserInteractionEnabled allowed for multiple components?,is there any specific requirement

Reply:
view heirarchy is situated on top of another so it is needed but childview's user interaction is not needed, will change it